src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -33,6 +33,11 @@ public static void Aggregate_AggregateException_Wraps_OperationCanceledException
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(0, (i, j) => j, i => i));
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(0, (i, j) => j, (i, j) => i, i => i));
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(() => 0, (i, j) => j, (i, j) => i, i => i));
+            Functions.AssertAggregateNotCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate((i, j) => j));
```

Comment:
Addendum - `SequenceEqual`'s behavior doesn't depend on source, and it's _all_ operations, so there are reductions available, but we'd still need the "output" tests to catch the behavior, at least.